app/src/main/java/layout/profileFragment.java

```diff
@@ -57,6 +57,10 @@ public View onCreateView(LayoutInflater inflater, final ViewGroup container,
 
         oldMoodListView = (ListView) view.findViewById(R.id.profileList);
 
+        if(FeelTripApplication.getThemeID() == R.style.CustomTheme_Light || FeelTripApplication.getThemeID() == R.style.CustomTheme_Dark) {
+            oldMoodListView.setBackgroundColor(FeelTripApplication.getBACKGROUNDCOLOR());
+        }
+
```

Comment:
Sets the background color for a dynamic coloring on custom theme, but this slice of code can be expanded to change the background to anything for our more complex themes as well.